### ISSUE TITLE
Presenting Aragon DAC at hackerspace 

### ISSUE THREAD

**Issue by krrisis** [aragon/aragondac-community#15]
Griff applied for a 'Blockchain Based Organizations NOW' space where we can ao talk about Aragon- more info here https://web3summit.com/program/hackerspace/

**Comment by GriffGreen** [comment]
Yep! So I am hopefully going to have a call with Josh from web3 right before the call tomorrow, so either I will have lots to talk about for how to set it up... or the details will come on Monday.
